.github/workflows/build.yml

```diff
@@ -30,6 +30,8 @@ jobs:
             BASE_IMAGE_NAME=${{ matrix.base_image_name }}
             BASE_IMAGE_TAG=${{ matrix.base_image_tag }}
             VCS_REF=${{ github.sha }}
+            ROS_DISTRO=''
+            ROS_APT_REPO_URL='http://packages.ros.org/ros'
```

Comment:
This needs to have both urls otherwise they're not usable for ros2

Reply:
So maybe we should use the list strategy, e.g.

```
for url in ROS_APT_REPO_URLS; do
  echo "deb ${url}/ubuntu $(lsb_release -sc) main" \
    > /etc/apt/sources.list.d/ros-latest.list	>> /etc/apt/sources.list.d/ros-latest.list
done
```

Note append >> rather than overwrite >

Reply:
Done.